python/lsst/pipe/base/connections.py

```diff
@@ -919,6 +919,56 @@ def adjustQuantum(self, inputs, outputs, label, data_id):
                 )
         return {}, {}
 
+    def getSpatialBoundsConnections(self) -> Iterable[str]:
+        """Return the names of regular input and output connections whose data
+        IDs should be used to compute the spatial bounds of this task's quanta.
+
+        The spatial bound for a quantum is defined as the union of the regions
+        of all data IDs of all connections returned here, along with the region
+        of the quantum data ID (if the task has spatial dimensions).
+
+        Notes
+        -----
+        The spatial bound is used to search for prerequisite inputs that have
+        skypix dimensions. The default implementation returns an empty
+        iterable, which is usually sufficient for tasks with spatial
+        dimensions, but if a task's inputs or outputs are associated with
+        spatial regions that extend beyond the quantum data ID's region, this
+        method may need to be overridden to expand the set of prerequisite
+        inputs found.
+
+        Tasks that do not have spatial dimensions that have skypix prerequisite
+        inputs should always override this method, as the default spatial
+        bounds otherwise cover the full sky.
+        """
+        return ()
+
+    def getTemporalBoundsConnections(self) -> Iterable[str]:
+        """Return the names of regular input and output connections whose data
+        IDs should be used to compute the temporal bounds of this task's
+        quanta.
+
+        The temporal bound for a quantum is defined as the union of the
+        timespans of all data IDs of all connections returned here, along with
+        the timespan of the quantum data ID (if the task has temporal
+        dimensions).
+
+        Notes
+        -----
+        The temproal bound is used to search for prerequisite inputs that are
+        calibration datasets. The default implementation returns an empty
+        iterable, which is usually sufficient for tasks with temporal
+        dimensions, but if a task's inputs or outputs are associated with
+        timespans that extend beyond the quantum data ID's timespan, this
+        method may need to be overridden to expand the set of prerequisite
+        inputs found.
+
+        Tasks that do not have temporal dimensions that have calibration
+        prerequisite inputs should always override this method, as the default
+        is that quanta are not temporally bounded.
```

Comment:
I find the phrase "the default is that quanta are not temporally bounded" to be confusing.  I'm not quite sure what this means?